Modules/Quotas/test/Quotas.Domain.Tests/QuotaTests.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using FluentAssertions;
+using Xunit;
+using Backbone.Modules.Quotas.Domain.Tests.Extensions;
+using Enmeshed.Tooling;
+
+namespace Backbone.Modules.Quotas.Domain.Tests;
+public class QuotaTests
+{
+    [Fact]
+    public void UpdateExhaustion_on_unexhausted_with_newUsage_under_max_quota_keeps_IsExhaustedUntil_null()
+    {
+        // Arrange
+        var definition = new TierQuotaDefinition(Aggregates.Metrics.MetricKey.NumberOfFiles, 50, QuotaPeriod.Hour);
```

Comment:
Why use such high numbers for Max when you can use lower numbers as well? :) So instead of 50 and 20, just use 2 and 1.